shared/components/pages/settings/user/login/login.rt.html

```diff
@@ -28,20 +29,24 @@
                 rt-class="{error: !this.paramValid('password')}" onChange="{this.updateInput.bind(this)}"
                 data-api_key="password"
                 value="{this.state.password}"
-                placeholder="{this.t('login.password')}">
+                placeholder="{this.t('login.password')}"/>
             </div>
             <div className="cc-component__auth-password-request">
                 <a onClick="{this.router.goToRouteByName.bind(this, 'ForgotPassword')}">
                   {this.t('login.password_request')}
                 </a>
             </div>
-            <div className="cc-component__auth-submit">
+            <div className="cc-component__auth-login">
                 <button type="submit" className="btn btn-default"
                 onClick="{this.submitLogin.bind(this)}">
                   {this.t('login.submit')}
                 </button>
             </div>
           </form>
+           <div className="cc-component__auth-submit">
```

Comment:
In my opinion it would be more clear and user-friendly if we had the two login options side-by-side.
Either that or just add a sentence saying 'Did you sign up with your Facebook Account?' above the button.